subsys/net/lib/nrf_cloud/src/nrf_cloud_agps.c

```diff
@@ -229,6 +236,11 @@ static int json_send_to_cloud(cJSON *const agps_request)
 	return err;
 }
 
+bool nrf_cloud_agps_request_in_progress(void)
+{
+	return request_active != 0;
+}
+
 int nrf_cloud_agps_request(const struct gps_agps_request request)
```

Comment:
do you know why this struct is passed in by value instead of a pointer?

Reply:
nope